File: VideoTube/DataAccess/Entities/Video.cs
using System;
namespace Entities.Entities;

public class Video
{
#nullable disable
    public Guid Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public long Likes { get; set; }
    public long Dislikes { get; set; }
    public string ThumbnailUrl { get; set; }
    public DateTime UloapDate { get; set; }
    public Guid UserID { get; set; }
    public string Url { get; set; }

    //foreign
    public virtual User Users { get; set; }
    public virtual ICollection<Comment> Comments { get; set; }
}

File: videoTubeFront/src/app/Models/Video.ts
import { Users } from "./Users";

export interface Video{
    id: string;
    title: string;
    description: string;
    likes: number;
    dislikes: number;
    thumbnailUrl: string;
    uloapDate: string;
    userId: string;
    url: string | '';
    users: Users;
}